tests/retif/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -171,28 +171,28 @@ $var = 1
     ? $someOtherReallyReallyLongVariable
     : $someOtherReallyReallyLongVariable
 )->call();
-$var = $test ?: true ? 1 : 2;
+$var = $test ?: (true ? 1 : 2);
 $var = $test
-    ?: $someOtherReallyReallyReallyReallyReallyReallyReallyReallyReallyReallyLongVariable
+    ?: ($someOtherReallyReallyReallyReallyReallyReallyReallyReallyReallyReallyLongVariable
         ? 1
-        : 2;
+        : 2);
 $var = $test
-    ?: true
+    ?: (true
         ? $someOtherReallyReallyReallyReallyReallyReallyReallyReallyReallyReallyLongVariable
-        : 2;
+        : 2);
 $var = $test
-    ?: true
+    ?: (true
         ? 1
-        : $someOtherReallyReallyReallyReallyReallyReallyReallyReallyReallyReallyLongVariable;
+        : $someOtherReallyReallyReallyReallyReallyReallyReallyReallyReallyReallyLongVariable);
```

Comment:
`falseExpr` always should have parens to avoid
```
echo true ?: (true ? 'foo' : 2); // Works as expected
echo true ?: true ? 'foo' : 2; // Works unexpected
```